includes/definitions/discovery/opengear.yaml

```diff
@@ -13,6 +13,7 @@ modules:
                     oid:
                         - ogCellModemTable
                         - ogEmdTemperatureTable
+                        - ogEmdHumidityTable
```

Comment:
Don't need this it is only used by one sensor, having it here will only make things slower.

Reply:
Actually, what are any of these doing here?

Reply:
yeah, please remove the whole pre-cache entry.

Reply:
ogEmdTemperatureTable isn't needed either...